.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,45 @@
+name: "CodeQL"
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ feat/refresh ]
+  pull_request:
+    branches: [ feat/refresh ]
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: true
+      matrix:
+        language: [ 'javascript', 'typescript' ]
```

```suggestion
        language: [ 'javascript' ]
```

The `typescript` is not a valid option in codeql. The `javascript` option will automatically run on both javascript and typescript files.

---

Yeah, makes sense. I added it by mistake. Thanks for pointing it out :)